content/03.methods.md

```diff
@@ -0,0 +1,36 @@
+We standardized (took z-scores of) each column of RNA sequencing and RPPA data; methylation data and mutational signatures data were left untransformed (beta values and mutation counts respectively), except in multi-omics experiments where all data types were standardized.
+For the RNA sequencing dataset, we used only the top 8,000 gene features by mean absolute deviation as predictors in our models, except in multi-omics experiments where all 15,639 genes were used.
+
+In order to remove missing values from the methylation datasets, we removed the 10 samples with the most missing values, then performed mean imputation for probes with 1 or 2 values missing.
+All probes with missing values remaining after sample filtering and imputation were dropped from the analysis.
+This left us with 20,040 CpG probes in the 27K methylation dataset, and 370,961 CpG probes in the 450K methylation dataset.
+For experiments where "raw" methylation data was used, we used the top 100,000 probes in the 450K dataset by mean absolute deviation for computational efficiency, and we used all of the 20,040 probes in the 27K dataset.
+For experiments where "compressed" methylation data was used, we used principal component analysis (PCA), as implemented in the `scikit-learn` Python library [@url:https://jmlr.csail.mit.edu/papers/v12/pedregosa11a.html], to extract the top 5,000 principal components from the methylation datasets.
+We also applied the beta-mixture quantile normalization (BMIQ) method [@doi:10.1093/bioinformatics/bts680] to correct for variability in signal intensity between type I and type II probes, but we observed that this had no effect on our results.
+A comparison of bias-corrected and uncorrected results is shown in (supplementary figure), and results presented in the main paper are uncorrected.
+
+To make a fair comparison in each of the experiments displayed in the results, we used the intersection of TCGA samples having measurements for all of the datasets being compared in that experiment.
+This resulted in 3 distinct sets of samples: 9,074 samples shared between expression and mutation data, 7,981 samples shared between expression/mutation/27K methylation/450K methylation, and 5,282 samples shared between expression/mutation/27K methylation/450K methylation/RPPA/mutational signatures.
+When we dropped samples between experiments as progressively more data types were added, we observed that the dropped samples were approximately uniformly distributed between cancer types.
+In other words, samples that were profiled for one data type but not another did not tend to come exclusively from one or a few cancer types.
+Exceptions included acute myeloid leukemia (LAML) which had no samples profiled in the RPPA data, and ovarian cancer (OV) which had only 8 samples with 450K methylation data.
+More detailed information on cancer type proportions profiled for each data type is provided in (the supplement).
```

Comment:
Probably would be a results figure since it comes from data analysis, but maybe could get away with a methods sup fig. In results it'd probably be from the first part of the results where you're introducing your findings.

Reply:
Makes sense - I was thinking supplementary figure because I'm not sure it's really part of the core story of the paper. Or, in other words, even if there were whole cancer types that were dropped at each filtering step, it wouldn't change our conclusions - we'd just drop those cancer types from our analysis completely, or note it here in the methods section and keep the results the same.

I don't feel too strongly about that though, I think we could put it in the overview figure. We can discuss when we meet today.